Plugins/Flow.Launcher.Plugin.Folder/Main.cs

```diff
@@ -146,14 +161,37 @@ private void InitialDriverList()
             }
         }
 
+        private void LoadEnvironmentStringPaths()
+        {
+            _envStringPaths = new Dictionary<string, string>();
+            
+            var specialPaths = 
```

Comment:
Maybe we could use `System.Environment.GetEnvironmentVariables()` here to get them dynamically loaded on each system?

Reply:
Apart from that I just tested it and it is working great! :)

Reply:
`GetEnvironmentVariables()` will return all sorts of variables, way more than the valid environment string paths like `%appdata%`. 

I considered loading them dynamically, but it seems there is no good way of doing it. `Environment.SpecialFolder` has some of them but not all (for example `temp` is missing) and it also has a lot of other special folders that do not correspond to valid environment-string paths.

Reply:
That said, we can add a couple more common ones like `%public%` and `%windir%` and cover 99% of use-cases...

Reply:
I am currently testing the usage of `System.Environment.GetEnvironmentVariables()`. We can check the results of that with `Directory.Exists()` and only add those to the dictionary. This also includes `%temp%` for example. In that way, we would also get user-defined environment variables, if I'm not mistaking.

Reply:
I could check-in my current test code for you to expand on it if you like.

Reply:
We could merge this and discuss your changes in a subsequent PR.

Reply:
Alright, I'll open another PR after this if you want.

Reply:
Thanks. We'll probably also need some reload logic if we go the env-var way, so let's keep them separate....